parsec/core/gui/main_window.py

```diff
@@ -553,19 +562,21 @@ def add_instance(self, start_arg: Optional[str] = None):
             except ValueError as exc:
                 show_error(self, _("TEXT_INVALID_URL"), exception=exc)
 
-        idx = self._get_login_tab_index()
-        if idx != -1:
-            self.switch_to_tab(idx)
-        else:
-            tab = self.add_new_tab()
-            tab.show_login_widget()
-            self.on_tab_state_changed(tab, "login")
-            self.switch_to_tab(self.tab_center.count() - 1)
-            idx = self.tab_center.count() - 1
+        # Do not open a new logging widget if the organisation is already opened
+        if (
+            action_addr
+            and isinstance(action_addr, BackendOrganizationFileLinkAddr)
+            and self.tab_center.count()
+        ):
+            self.go_to_file_link(action_addr)
+            return
+
+        self.switch_to_login_tab()
 
         self.show_top()
```

Comment:
We don't do `self.show_top()` in case of file link addr and logged in organization ? wouldn't this cause some issues if te parsec gui is in background or minimized when clicking on the link ?

Reply:
It's a mistake, `show_top() ` is now always called (for all kind of parsec link)